SDK Showcase/SDK Showcase/SDK_ShowcaseApp.swift

```diff
@@ -1,7 +1,9 @@
+//  Copyright © 2025 Onewelcome Mobile Identity. All rights reserved.
+
 import SwiftUI
 
 @main
-struct ExampleAppV2App: App {
+struct SDK_ShowcaseApp: App {
```

Comment:
How about renaming this?
![Zrzut ekranu 2025-01-21 o 12 29 00](https://github.com/user-attachments/assets/09587396-4e20-4018-a3c3-12295b41bfac)

Reply:
this is removed from the project after the name change., please remove the repo and fetch it again